Tensile/Tests/integration/test_integration.py

```diff
@@ -0,0 +1,127 @@
+import os, subprocess, shlex, random, pytest
+from Tensile import ClientWriter, LibraryIO, Common
+from Tensile.SolutionStructs import ProblemType, Problem
+
+# 1. Call TensileCreateLibrary
+# 2. Get client instance from ClientExecutable()
+# 3. Write some problem types/sizes to ClientParameters with writeClientConfig()
+# 4. Run client with small subset of GEMM problems
+# $ pytest Tensile/Tests -m integration --capture=tee-sys --builddir /tmp/pytest
+
+def getLogicFileDir(baseDir, schedule):
```

Comment:
The entire rocBLAS zip package is only 7.3 mb.  Maybe that's a simpler way to get these files.

Reply:
that's a great suggestion